tensorly/regression/cp_plsr.py

```diff
@@ -283,3 +317,18 @@ def fit_transform(self, X, Y):
             Return `x_scores` if `Y` is not given, `(x_scores, y_scores)` otherwise.
         """
         return self.fit(X, Y).transform(X, Y)
+
+    def score(self, X, Y):
+        """Calculate the R^2 of prediction on X compared to the ground truth Y provided.
+
+        Parameters
+        ----------
+        X : ndarray
+            tensor data of shape (n_samples, N1, ..., NS), same dimension as the X
+            in self.fit() all except the first dimension
+        Y : 2D-array of shape (n_samples, n_predictions)
+            the ground truth labels associated with each sample
+        """
+        from ..metrics.regression import R2_score
```

Comment:
Is there a reason this is imported just at the function level?

Reply:
Oh this is just mimicking how [sklearn did it](https://github.com/scikit-learn/scikit-learn/blob/9aaed498795f68e5956ea762fef9c440ca9eb239/sklearn/base.py#L720).